test/qa/lib/daemon.py

```diff
@@ -150,4 +150,4 @@ def is_peer_running(ssh_client: ssh.Ssh) -> bool:
     except:
```

Comment:
[minor] Cannot add comment on the function name, but it is a bit confusing. maybe in future can be changed to something like is daemon/service running for other peer?

Reply:
Hey, probably you are right :smile: However, changes like this may not be limited to only that function name, it may be, that other related functions will also need renaming, so I think this may need a separate task. Maybe even module itself needs refactoring. However, this is sort of getting out of the scope of my current task